docs/contributing/structure-guide.md

```diff
@@ -176,6 +184,10 @@ The `@return` for simple RestActions is as follows:
 ```
 
 ### Example Template
+
+!!! note inline end
```

Comment:
I don't know if we should really use an inline admonition here? This is quite a big code block, and having an inline admonition there really reduces its size by a lot, making you scroll down to the bottom of the code block to use the horizontal scroll block. That's why I didn't inline it and just put it at the end.

Reply:
Kinda agree. An alternative could be to use codeblock annotations to point out important sections.

Reply:
Yeah, that'd be nice. They're not as obvious as the admonitions though, so I suppose that'd be dependent on how important the note is then

Reply:
I did switch it with a code annotation, which imo looks decent enough and not too massive here.